### PR TITLE
Fixes typo for GraphQL, Adds GraphQL test

### LINKED ISSUE #381: GraphaQL demo is not working
I tried to test the GraphQL addition on test-start.microprofile.io. I found the following issues:
1. the Readme on service-a is not updated with the additional graphql instructions
2. after choosing GraphQL box, the code was generated but the endpoint was not available. We need to update the index.html to include the endpoint of graphQL as well.

### PR DESCRIPTION
**Pull request by Karm**
Fixes #381 

* fixes typo
* adds GraphQL test

### CODE REVIEW

File: src/it/java/org/eclipse/microprofile/starter/utils/MPSpecGET.java [hunk 1]
```diff
@@ -61,7 +61,10 @@
     //}),
     REST_CLIENT(new String[][]{
             new String[]{"data/client/test/parameterValue=xxx", "Processed parameter value 'parameterValue=xxx'"}
-    });
+    }),
+    GRAPH_QL(new String[][]{
+            new String[]{"data/client/test/parameterValue=xxx", "Wolverine"}
```
Comment (Emily-Jiang) [inline src/it/java/org/eclipse/microprofile/starter/utils/MPSpecGET.java:66]: I don't think this this is the valid url for graphQL though

Comment (Karm) [reply src/it/java/org/eclipse/microprofile/starter/utils/MPSpecGET.java:66]: It is not. A stray thing....   this is a draft :-)

**Comment by Karm** [comment]
@Emily-Jiang Redy for review

**Review by Emily-Jiang — APPROVED** [review]
LGTM! Thanks @Karm !

**Review by andymc12 — CHANGES_REQUESTED** [review]
Hi @Karm - thanks for fixing this.  Instead of changing the "tpl" changes to use "mp_graph_ql", could we change the `StandaloneMPSpec` to use `graphql` (with no underscore) instead?  There really shouldn't be an underscore between "graph" and "ql", and I think it might cause confusion - especially when searching the source.

Also, fwiw, this will only expose the app's GraphQL schema.  It might be nice to add something like this to the index.html.tpl so that users can see the UI to issue their own queries/mutations/etc. from 
```
[# th:if="${mp_graphql}"]
<h3>GraphQL</h3>
<a href="graphql/schema.graphql" target="_blank" >GraphQL schema</a> <br/>
[/]
```

to:
```
[# th:if="${mp_graphql}"]
<h3>GraphQL</h3>
<a href="graphql/schema.graphql" target="_blank" >GraphQL schema</a> <br/>
<a href="graphql-ui" target="_blank" >GraphQL UI</a> <br/>
[/]
```

Thanks again!

File: src/it/java/org/eclipse/microprofile/starter/utils/Whitelist.java [hunk 2]
```diff
@@ -36,8 +36,11 @@
             Pattern.compile(".*com.hazelcast.nio.tcp.TcpIpConnectionErrorHandler.*"),
     }),
     LIBERTY("liberty", new Pattern[]{
-        Pattern.compile(".*FrameworkEvent ERROR.*"),
-        Pattern.compile(".*CWWKE0701E.*"),
+            Pattern.compile(".*FrameworkEvent ERROR.*"),
+            Pattern.compile(".*CWWKE0701E.*"),
+            // An exception occurred while stopping the application liberty. The exception message was: java.lang.NoClassDefFoundError: com/ibm/ws/threading/internal/ImmediateFutureImpl
+            Pattern.compile(".*CWWKZ0010E:.*"),
+            Pattern.compile(".*Could not load service class com.ibm.ws.io.smallrye.graphql.component.GraphQLExtension.*")
```
Comment (Emily-Jiang) [inline src/it/java/org/eclipse/microprofile/starter/utils/Whitelist.java:43]: @andymc12 when will the warning be fixed?

**Comment by Karm** [comment]
@andymc12 O.K., I can't see why not. All GraphQL things are renamed so as there is no underscore. The link to the UI is added.

**Comment by Karm** [comment]
Ready for review.

**Comment by Karm** [comment]
Fixed API test

**Comment by Karm** [comment]
Whitelisting more and more Liberty intermittent error messages in the log.

File: src/it/java/org/eclipse/microprofile/starter/utils/Whitelist.java (hunk 2 above)
Comment (andymc12) [reply src/it/java/org/eclipse/microprofile/starter/utils/Whitelist.java:45]: Good question - I hadn't noticed this before.  I'll open an issue and look into why the warning is occurring.  Thanks for mentioning it.

**Review by andymc12 — APPROVED** [review]
The changes look great!  Thanks @Karm!

**Pull request merged**